src/test/java/teammates/test/cases/ui/XlsxFileDownloadResultTest.java

```diff
@@ -0,0 +1,90 @@
+        assertEquals(1, download.getWorkBook().getSheetAt(0).getRow(1).getLastCellNum());
+    }
+
+    @Test
+    public void testTwoWordsContentXslFileDownloadResult() {
+        String fileContent = "one,two";
+        XlsxFileDownloadResult download = new XlsxFileDownloadResult(
+                "/page/instructorFeedbackResultsDownload",
+                null, null, "Normal name",
+                fileContent, Const.FeedbackSessionResultsDownloadTypes.XLSX);
+        assertEquals(1, download.getWorkBook().getSheetAt(0).getLastRowNum());
+        assertEquals(2, download.getWorkBook().getSheetAt(0).getRow(1).getLastCellNum());
+
+    }
+
+    @Test
+    public void testTwoRowsContentXslFileDownloadResult() {
+        String fileContent = "one,two" + Const.EOL + "three, four";
+        XlsxFileDownloadResult download = new XlsxFileDownloadResult(
+                "/page/instructorFeedbackResultsDownload",
+                null, null, "Normal name",
+                fileContent, Const.FeedbackSessionResultsDownloadTypes.XLSX);
+        assertEquals(2, download.getWorkBook().getSheetAt(0).getLastRowNum());
+        assertEquals(2, download.getWorkBook().getSheetAt(0).getRow(1).getLastCellNum());
+        assertEquals(2, download.getWorkBook().getSheetAt(0).getRow(2).getLastCellNum());
+    }
+
+    @Test
+    public void testWordsWithCommaContentXslFileDownloadResult() {
+        String fileContent = "one,two" + Const.EOL + "three, four" + Const.EOL
+                + Sanitizer.sanitizeForCsv("five, six");
+        XlsxFileDownloadResult download = new XlsxFileDownloadResult(
+                "/page/instructorFeedbackResultsDownload",
+                null, null, "Normal name",
+                fileContent, Const.FeedbackSessionResultsDownloadTypes.XLSX);
+        assertEquals(3, download.getWorkBook().getSheetAt(0).getLastRowNum());
+        assertEquals(2, download.getWorkBook().getSheetAt(0).getRow(1).getLastCellNum());
+        assertEquals(2, download.getWorkBook().getSheetAt(0).getRow(2).getLastCellNum());
+        // Third row should only have one cell since it a single word with a
+        // comma included.
+        assertEquals(1, download.getWorkBook().getSheetAt(0).getRow(3).getLastCellNum());
+
+    }
+
+    @Test
+    public void testQuestionFormattingChanges() throws IOException {
+        String fileContent = "one,two" + Const.EOL + "Question 1,"
+                + Sanitizer.sanitizeForCsv("Hi What do you think about Calculus, Differential Classes ?");
+        XlsxFileDownloadResult download = new XlsxFileDownloadResult(
+                "/page/instructorFeedbackResultsDownload",
+                null, null, "Normal name",
+                fileContent, Const.FeedbackSessionResultsDownloadTypes.XLSX);
+        assertEquals(2, download.getWorkBook().getSheetAt(0).getLastRowNum());
+        assertEquals(2, download.getWorkBook().getSheetAt(0).getRow(1).getLastCellNum());
```

Comment:
Let's extract the repeated `download.getWorkBook().getSheetAt(0)` to a local variable to improve readability.